### PR TITLE
FIX:  KIT reference channels

### PR DESCRIPTION
**Pull request by christianbrodbeck**
For UMD files from system ID 52 reference channels seem to be labeled differently. Is it worth adding another small raw file for the tests?

The actual Fix is lines 608-15, everything below is just refactoring I did while debugging and could be reverted.

CC @teonbrooks

### CODE REVIEW

File: mne/io/kit/kit.py [hunk 1]
```diff
@@ -680,27 +684,19 @@ def get_kit_info(rawfile):
 
         # Creates a list of dicts of meg channels for raw.info
         logger.info('Setting channel info structure...')
-        ch_names = {}
-        ch_names['MEG'] = ['MEG %03d' % ch for ch
-                           in range(1, sqd['n_sens'] + 1)]
-        ch_names['MISC'] = ['MISC %03d' % ch for ch
-                            in range(1, sqd['nmiscchan'] + 1)]
```
Comment (agramfort) [inline mne/io/kit/kit.py]: why changing this?

Comment (christianbrodbeck) [inline mne/io/kit/kit.py]: PyCharm suggested refactors and what I found clearer (more red than green :) ), but I can revert if you want, there is no change in function.

**Comment by teonbrooks** [comment]
thanks @christianbrodbeck. I need to dig through the c++ code to check why the `sens_type` was set to 257. I think I got the check from there. It's on my hard drive, which is currently somewhere in my luggage ;)

**Comment by teonbrooks** [comment]
I just did a search and it looks like the assignment of `sens_type` to `sqd['i']` isn't even used and the coordinate of the reference channels are defined with just zeros. I think it was just a way to filter for the ref channels.

**Comment by christianbrodbeck** [comment]
> I just did a search and it looks like the assignment of sens_type to sqd['i'] isn't even used and the coordinate of the reference channels are defined with just zeros. I think it was just a way to filter for the ref channels.

Thanks! Good to know, then that modification should be safe. The problem was just that if a channel had sens_type 0 it was skipped, leading to an erraneous sensor count later. 

Do we know the coordinates of the reference channels, which is set to zero currently? I assume if it were in the file you would have used it...

**Comment by teonbrooks** [comment]
> Do we know the coordinates of the reference channels, which is set to zero currently? I assume if it were in the file you would have used it...

I don't know if the position of the reference is written in the file. when I have a chance I can dig into the old code. it seems that they are just set to zero origin.

**Comment by christianbrodbeck** [comment]
@agramfort some people here at UMD expressed interest in looking at older data, would it be a lot of work to do a maintenance release if I cherry-picked these commits into the maint/0.13 branch?

**Comment by agramfort** [comment]
yes we can do a maint release

PR on maint branch welcome

**Pull request merged**